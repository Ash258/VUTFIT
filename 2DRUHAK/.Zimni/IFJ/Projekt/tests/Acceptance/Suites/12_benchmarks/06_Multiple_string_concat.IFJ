' Multiple string concatenations

scope
	dim buffer as string
	dim count as integer = 42
	do while count > 0
        buffer = !"" + buffer + !" barman" + !"" ' for price and optimalization
        count = count - 1
	loop
	print buffer;
end scope
