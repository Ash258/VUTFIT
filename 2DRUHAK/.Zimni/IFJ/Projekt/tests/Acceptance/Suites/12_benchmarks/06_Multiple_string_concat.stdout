 barman barman barman barman barman barman barman barman barman barman barman barman barman barman barman barman barman barman barman barman barman barman barman barman barman barman barman barman barman barman barman barman barman barman barman barman barman barman barman barman barman barman